src/elements/common/header/__tests__/Header.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { HeaderBase as Header } from '../Header';
+
+describe('elements/common/header/Header', () => {
+    const intl = {
+        formatMessage: jest.fn().mockImplementation(message => message.defaultMessage),
+    };
+
+    const getWrapper = props => shallow(<Header intl={intl} {...props} />);
+
+    test('renders Logo component when isHeaderLogoVisible is passed', () => {
+        const wrapper = getWrapper({ isHeaderLogoVisible: true });
+        expect(wrapper.find('Logo').exists()).toBe(true);
+    });
+
+    test('renders matching values for aria-label and placeholder attributes', () => {
+        const wrapper = getWrapper();
+        const inputProps = wrapper.find('[data-testid="be-search-input"]').props();
+        expect(inputProps['aria-label']).toBe(inputProps.placeholder);
```

Comment:
minor: you're using using `inputProps['aria-label']`, but `inputProps.placeholder`. They're both fine, but I think we should be consistent

Reply:
yeah this was an eslint thing. it prefers dot notation but it can't be used for the aria-label

Reply:
oh that's interesting, thanks for the info